### PR TITLE
feat(redirection): adds mqtt start and stop events for redirection

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [x] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

## What are you changing?
Adds MQTT start and stop events for redirection sessions

## Anything the reviewer should know when reviewing this PR?
Hold until after 2.2 release

### CODE REVIEW

File: src/utils/wsRedirect.ts [hunk 1]
```diff
@@ -46,6 +47,7 @@ export class WsRedirect {
 
     // We got a new web socket connection, initiate a TCP connection to the target Intel AMT host/port.
     logger.debug(`${messages.REDIRECT_OPENING_WEB_SOCKET} to ${params.host}: ${params.port}.`)
+    MqttProvider.publishEvent('success', ['handleConnection'], messages.REDIRECTION_SESSION_STARTED)
```
Comment (Walt-Intel) [inline src/utils/wsRedirect.ts:50]: Can this call fail? Is any error handling needed?

File: src/utils/wsRedirect.ts [hunk 2]
```diff
@@ -76,6 +78,7 @@ export class WsRedirect {
     logger.debug(`${messages.REDIRECT_CLOSING_WEB_SOCKET} to ${params.host}: ${params.port}.`)
     if (this.websocketFromDevice) {
       this.websocketFromDevice.CloseChannel()
+      MqttProvider.publishEvent('success', ['handleClose'], messages.REDIRECTION_SESSION_ENDED)
```
Comment (Walt-Intel) [inline src/utils/wsRedirect.ts:81]: Can this call fail? Is any error handling needed?

**Review by Walt-Intel — APPROVED** [review]
Looks good to me.

File: src/utils/wsRedirect.ts (hunk 1 above)
Comment (matt-primrose) [reply src/utils/wsRedirect.ts:50]: If a MQTT broker is not configured/missing, the call will silently fail without impacting the rest of the code

File: src/utils/wsRedirect.ts (hunk 2 above)
Comment (matt-primrose) [reply src/utils/wsRedirect.ts:81]: If a MQTT broker is not configured/missing, the call will silently fail without impacting the rest of the code

**Pull request merged**